doc/source/cluster/kubernetes/user-guides/pod-command.md

```diff
@@ -1,18 +1,69 @@
 (kuberay-pod-command)=
 
 # Specify container commands for Ray head/worker Pods
-You can execute commands on the head/worker pods at two timings:
 
-* (1) **Before `ray start`**: As an example, you can set up some environment variables that will be used by `ray start`.
+KubeRay generates a `ray start` command for each Ray Pod.
```

Comment:
It may be helpful to include an example of the full command here, just so users know what it looks like